### ISSUE TITLE
public item note disappears when item checked out

### ISSUE THREAD

**Issue by theLinkResolver** [ualbertalib/discovery#1523]
**Describe the bug**
Notes for public display that are attached to a specific item (commonly called "public notes") do not display when the item is checked out.

**To Reproduce**
Currently, look at this record: https://search.library.ualberta.ca/catalog/8501416

**Expected behavior**
Public item notes should always display.

**Screenshots**
This shows the note not displaying.
![cap](https://user-images.githubusercontent.com/14913579/53831889-248ba100-3f43-11e9-8b7d-90edbfec38c4.jpg)

Here's the note displaying (different record).
![cap2](https://user-images.githubusercontent.com/14913579/53831926-3705da80-3f43-11e9-8b75-a6fd6b9255e9.jpg)

**Comment by seanluyk** [comment]
Good catch @theLinkResolver  - looks like the [current logic](https://github.com/ualbertalib/discovery/blob/b7afda58d0c3ca8db501bf05d8697a3fd951b3e8/app/services/symphony_service.rb#L66) is to populate the due date only when an item is checked out.
